psl/psl/tests/validation/composite_types/index_attributes_on_composite_types.prisma

```diff
@@ -60,3 +60,15 @@ model B {
 // [1;94m15 | [0m    @@index([other, field])
 // [1;94m16 | [0m    [1;91m@@unique([content, rank])[0m
 // [1;94m   | [0m
+// [1;91merror[0m: [1mAttribute not known: "@id".[0m
+//   [1;94m-->[0m  [4mschema.prisma:7[0m
+// [1;94m   | [0m
+// [1;94m 6 | [0mtype A {
+// [1;94m 7 | [0m    pk String [1;91m@id[0m
+// [1;94m   | [0m
+// [1;91merror[0m: [1mAttribute not known: "@unique".[0m
```

Comment:
Not for this time, but I think error message could now be changed to say that `@id` and friends are not available on composite types.